site/ja/index.html

```diff
@@ -105,6 +105,7 @@
                     <ul class="dropdown-menu">
                         <li><a href="../index.html">English</a></li>
                         <li><a href="">Japanese</a></li>
+						<li><a href="../pt-br/index.html">Portuguese</a></li>
```

Comment:
This should change to: Português-Brasil
Also please update the copyright to say

> and others